spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -1726,7 +1726,7 @@ def assert_specific_cloud_database_flavor
       :ems_ref => "db.r3.large",
       :type    => "ManageIQ::Providers::Amazon::CloudManager::CloudDatabaseFlavor",
       :cpus    => 2,
-      :memory  => 16106127360
+      :memory  => 16374562816
```

Comment:
NOTE the previous value was `15GiB` where the new one has `15.25GiB` I forget where I pulled the old list from but it either didn't include fractional memory or this has been updated since